src/plugins/ResizeReorder/ResizeCell.js

```diff
@@ -64,12 +62,15 @@ class ResizeCell extends React.PureComponent {
           height={this.props.height}
           resizerLineHeight={this.context.tableHeight}
           onColumnResizeEnd={this.props.onColumnResizeEnd}
+          onColumnReorderEnd={this.props.onColumnReorderEnd}
```

Comment:
why we need  this  here ?

Reply:
To correct the functionality of resize and reorder cell combined example .Because in that example we are firstly rendering the Reorder Plugin and than it's child which is Resize plugin .So there is one more div which is being added which is the div of reorder cell and we are providing the styling in this div only which is also adjusting the cell from left .So, if our resize cell is being rendered after reorder cell than we have to adjust the left of resizer knob differently .The code of styling this resizer knob is in ResizerKnobFunction.js